tests/system/tostring.nim

```diff
@@ -108,13 +104,22 @@ bar(nilstring)
 static:
   stringCompare()
 
-# bug 8847
+# issue #8847
```

Comment:
https://nim-lang.github.io/Nim/contributing.html

> Always refer to a GitHub issue using the following exact syntax: bug #1234 as shown above, so that it's consistent and easier to search or for tooling. [...]